packages/elements/package.json

```diff
@@ -20,9 +20,10 @@
   "module": "dist/elements.esm.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "build:prod": "concurrently \"rimraf dist && cross-env NODE_ENV=production tsdx build --format=cjs,esm,umd && rollup -c\" \"rimraf out && build-storybook -o out\"",
+    "build:prod": "concurrently \"rimraf dist && cross-env NODE_ENV=production tsdx build --format=cjs,esm,umd && rollup -c\" \"rimraf public/dist && build-storybook -o public/dist\"",
     "lint": "concurrently \"tsc --noEmit\" \"eslint --cache --ext=ts,tsx,js src\"",
-    "release:prod": "node ../../scripts/release/release-npm.js elements",
+    "release:prod": "node ../../scripts/release/release-npm.js elements && node ../../scripts/release/release-prod.js elemements reapit-elements-prod",
```

Comment:
A typo, should be `elements` instead of `elemements`

Reply:
@vuhuucuong thank you